graylog2-web-interface/src/views/components/messagelist/MessageFields/MessageFieldEditMode.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import React, { useContext, useCallback } from 'react';
+import styled, { css } from 'styled-components';
+
+import type FieldType from 'views/logic/fieldtypes/FieldType';
+import type { Message } from 'views/components/messagelist/Types';
+import { MessageFieldName, MessageFieldValue } from 'views/components/messagelist/MessageField';
+import MessageFavoriteFieldsContext from 'views/components/contexts/MessageFavoriteFieldsContext';
+import CommonFavoriteIcon from 'views/components/common/CommonFavoriteIcon';
+
+type Props = {
+  fieldName: string;
+  fieldType: FieldType;
+  message: Message;
+  value: any;
+  isFavorite: boolean;
+};
+
+const DefinitionDescription = styled.dd(
+  ({ theme }) => `
+  font-family: ${theme.fonts.family.monospace};
+`,
+);
+
+const FieldValueContainer = styled.div(
+  () => css`
+    display: flex;
+    flex-direction: column;
+  `,
+);
+
+const Container = styled.div(
+  ({ theme }) => css`
+    display: flex;
+    gap: ${theme.spacings.xs};
+    align-items: center;
+  `,
+);
+
+const MessageFieldEditMode = ({ fieldName, fieldType, message, value, isFavorite }: Props) => {
+  const { removeFromFavoriteFields, addToFavoriteFields } = useContext(MessageFavoriteFieldsContext);
+  const onFavoriteToggle = useCallback(() => {
+    const clb = isFavorite ? removeFromFavoriteFields : addToFavoriteFields;
+
+    return clb(fieldName);
+  }, [addToFavoriteFields, fieldName, isFavorite, removeFromFavoriteFields]);
+
+  const favTitle = isFavorite ? `Remove ${fieldName} from favorites` : `Add ${fieldName} to favorites`;
+
+  return (
+    <Container>
+      <CommonFavoriteIcon isFavorite={isFavorite} title={favTitle} onClick={onFavoriteToggle} />
+      <FieldValueContainer>
+        <b>
+          <MessageFieldName message={message} fieldName={fieldName} fieldType={fieldType} />
```

Comment:
The field action menu seems to be hidden behind the modal.

<img width="531" height="230" alt="Image" src="https://github.com/user-attachments/assets/63274646-5ac7-4f01-8f40-d8c5380940cf" />